src/api/api.ts

```diff
@@ -205,9 +205,7 @@ export const axiosPutUserSubjects = async (
   subjects: Subject[]
 ) => {
   const url = `${apiURL}/user/${id}/subjects`;
-  await axios
-    .put(url, subjects, { headers: { token } })
-    .catch(logError('putUserSubjects'));
```

Comment:
Can we keep this logging on DEV (consistent with all other API requests), or is there a reason to remove it?

Reply:
Oh, hmm, strange; seems like that was removed when I merged master into it